[Fundamentals] Abuse reports (#14475)
cloudflare/cloudflare-docs: content/fundamentals/setup/account/abuse-reports.md

---
pcx_content_type: concept

> title: Abuse reports
> weight: 5
> ---
> 
> # Abuse reports

Reviewer: ```suggestion
title: Submit abuse reports
weight: 5
---

# Submit abuse reports
```
I'd start with a verb, like other pages in the same section

Cloudflare permits any interested party to submit abuse reports directly to Cloudflare via [abuse.cloudflare.com](https://abuse.cloudflare.com/). 

> Abuse reports may be submitted for suspected copyright or trademark infringement, illegal, or harmful content (e.g., child sex abuse materials), technical abuse (e.g., phishing or malware), or other reasons. 

Reviewer: ```suggestion
Abuse reports may be submitted for suspected copyright or trademark infringement, illegal, or harmful content (for example, child sex abuse materials), technical abuse (for example, phishing or malware), or other reasons. 
```

You may receive an abuse report from our Trust & Safety team if an abuse report identifies a URL for a domain associated with your Cloudflare account. If you do not provide or monitor an abuse contact, Cloudflare will send abuse reports to your hosting provider.

Our Trust & Safety team sends abuse reports to the domain owner or the abuse point of contact on your account.

> In order to assist with timely resolution and avoid potential service interruptions:

Reviewer: ```suggestion
To assist with timely resolution and avoid potential service interruptions:
```
